extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/exec/MSQWindowTest.java

```diff
@@ -2323,4 +2326,23 @@ public void testReplaceWithPartitionedByDayOnWikipedia(String contextName, Map<S
                      )))
                      .verifyResults();
   }
+
+  @MethodSource("data")
+  @ParameterizedTest(name = "{index}:with context {0}")
```

Comment:
what's the point of parameterizing this test? do those parameter changes relevant for this test?

Reply:
I think we parameterize these tests just for better naming?

The `data` itself consists of a single element:
```
public static Collection<Object[]> data()
  {
    Object[][] data = new Object[][]{
        {DEFAULT, DEFAULT_MSQ_CONTEXT}
    };

    return Arrays.asList(data);
  }
```

All the tests in this file are the same, so just keeping it consistent with them.